pkg/webhook/managedresource/validatingadmissionpolicy.go

```diff
@@ -42,58 +42,30 @@ func mutateValidatingAdmissionPolicy(vap *admv1.ValidatingAdmissionPolicy, isHub
 				{
 					RuleWithOperations: admv1.RuleWithOperations{
 						Rule: admv1.Rule{
```

Comment:
looks like the "isHub" parameter is no longer used, I am surpised that lint didn't complain abou it.